libpod/oci_conmon_linux.go

```diff
@@ -1335,7 +1330,23 @@ func (r *ConmonOCIRuntime) sharedConmonArgs(ctr *Container, cuuid, bundlePath, p
 
 // startCommandGivenSelinux starts a container ensuring to set the labels of
 // the process to make sure SELinux doesn't block conmon communication, if SELinux is enabled
-func startCommandGivenSelinux(cmd *exec.Cmd) error {
+func startCommandGivenSelinux(cmd *exec.Cmd, ctr *Container) error {
+	// Make sure to unset the NOTIFY_SOCKET and reset if aftewards if needed.
```

Comment:
You will save Dan time in a future spellcheck PR, if you spell this as 'afterwards' (with an 'r').

Reply:
Thanks! Updated locally, if there's more to change, I'll push it.

Reply:
do we need to fix this before merge?

Reply:
Repushed.